test/stringify.test.ts

```diff
@@ -156,14 +156,14 @@ describe('stringify', () => {
       title: 'Test Recording',
       steps: [
         {
-          type: 'scroll' as const,
+          type: StepType.Scroll as const,
```

Comment:
would we still need `as const` here and in other places?

Reply:
It was surprising to me as well, but yes, still needed.

![image](https://user-images.githubusercontent.com/16611980/194070982-782271cc-672c-496b-bc4e-22d2fe2ff2e6.png)